src/App.tsx

```diff
@@ -1,73 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+
+import { Autocomplete } from './components/Autocomplete';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handleSelectedPerson = (person: Person | null) => {
+    setSelectedPerson(person);
+  };
 
   return (
     <div className="container">
       <main className="section is-flex is-flex-direction-column">
         <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
+          {selectedPerson
+            ? `${selectedPerson?.name} (${selectedPerson?.born} - ${selectedPerson?.died})`
+            : 'No selected person'}
```

Comment:
The conditional rendering of `selectedPerson` is correct, but using optional chaining (`?.`) is unnecessary here since `selectedPerson` is already checked for truthiness. You can safely use `selectedPerson.name`, `selectedPerson.born`, and `selectedPerson.died` without optional chaining.